common/mount.py

```diff
@@ -433,7 +440,9 @@ def __init__(self,
 
         self.config = cfg or config.Config()
 
-        self.profile_id = profile_id if profile_id is not None else self.config.currentProfile()
+        self.profile_id = (
+            profile_id if profile_id is not None else self.config.currentProfile()
+        )
```

Comment:
Smells like `black`.
```suggestion
        self.profile_id = profile_id or self.config.currentProfile()
```